bl-plugins/google/plugin.php

```diff
@@ -16,14 +16,14 @@ public function form()
 		global $Language;
 
 		$html  = '<div>';
-		$html .= '<label for="jsgoogle-site-verification">'.$Language->get('Google Webmasters tools').'</label>';
-		$html .= '<input id="jsgoogle-site-verification" type="text" name="google-site-verification" value="'.$this->getDbField('google-site-verification').'">';
+		$html .= '<label>'.$Language->get('Google Webmasters tools').'</label>';
+		$html .= '<input type="text" name="google-site-verification" value="'.$this->getDbField('google-site-verification').'">';
```

Comment:
Hi, why you remove the id and the for from the label ?

Reply:
Hi! Well, there is no need to have a link between <label> and <input> because no expected actions. It works fine without them. But I can revert this changes to follow strict HTML standards.